docs/dev-guides/destroy_deployment.md

```diff
@@ -163,7 +159,7 @@ First, navigate to the AWS EC2 console - this step should be identical for both
 
 ### **Delete EBS snapshots**
 1. Click the _Snapshots_ link in the sidebar menu
-    1. Filter by osl-stackname: `<deployment_name>`
+    1. Filter by `osl-billing`: `<deployment_name>`
```

Comment:
By default, the cost allocation tag name is `osl-billing`. But there is no guarantee that it will be that value. I'm thinking that replacing `osl-billing` with <cost allocation tag> would be more appropriate.